(release) Q*cert 1.3.0 with Coq 8.8.2, improved JavaScript backend…

jeromesimeon opened the pull request:
- Support for Coq 8.8.2 and OCaml 4.07.1
- Additional work on JavaScript backend
- Additions to built-in operators
- Bug fixes (Wrong order in bag difference)

Signed-off-by: Jerome Simeon <jeromesimeon@me.com>

palmskog commented:
Since OPAM 2 keeps track of installed files, can you please remove the `remove` and `flags: light-uninstall`?

Also, can you please add some metadata in `tags` as per [this example](https://github.com/coq/opam-coq-archive#website-and-opam-metadata)? In particular, use `Qcert` for `logpath` and include the date of the release.

jeromesimeon commented:
> Since OPAM 2 keeps track of installed files, can you please remove the `remove` and `flags: light-uninstall`?
> 
> Also, can you please add some metadata in `tags` as per [this example](https://github.com/coq/opam-coq-archive#website-and-opam-metadata)? In particular, use `Qcert` for `logpath` and include the date of the release.

Thanks for pointing those out. I've updated the opam metadata. I also noticed the new directory name was wrong so I fixed that in the pull request too.

jeromesimeon commented:
Thanks!
